plugin-server/tests/worker/ingestion/event-pipeline/emitToBufferStep.test.ts

```diff
@@ -43,19 +43,17 @@ beforeEach(() => {
         hub: {
             CONVERSION_BUFFER_ENABLED: true,
             BUFFER_CONVERSION_SECONDS: 60,
-            db: { fetchPerson: jest.fn().mockResolvedValue(existingPerson) },
-            eventsProcessor: {
-                produceEventToBuffer: jest.fn(),
-            },
+            db: { fetchPerson: jest.fn().mockResolvedValue(existingPerson), addEventToBuffer: jest.fn() },
+            eventsProcessor: {},
         },
     }
 })
 
 describe('emitToBufferStep()', () => {
-    it('calls `produceEventToBuffer` if event should be buffered, stops processing', async () => {
+    it('calls `addEventToBuffer` if event should be buffered, stops processing', async () => {
         const response = await emitToBufferStep(runner, pluginEvent, () => true)
 
-        expect(runner.hub.eventsProcessor.produceEventToBuffer).toHaveBeenCalledWith(pluginEvent)
+        expect(runner.hub.db.addEventToBuffer).toHaveBeenCalledWith(pluginEvent, expect.any(DateTime))
```

Comment:
we should be asserting the datetime here as well given the business logic in the main code.